qiskit_dynamics/signals/signals.py

```diff
@@ -272,6 +272,10 @@ class DiscreteSignal(Signal):
     The envelope is specified by an array of samples ``s = [s_0, ..., s_k]``, sample width ``dt``,
     and a start time ``t_0``, with the envelope being evaluated as
     :math:`f(t) =` ``s[floor((t - t0)/dt)]``.
+    If the signal is sampled outside of its defined sample list, it will return a 0.0. So if `
```

Comment:
The comment that the envelope is `0.0` outside of the interval `start_time` and `start_time + dt * len(samples)` should be merged into the preceding sentence. E.g.

```
    The envelope is specified by an array of samples ``s = [s_0, ..., s_k]``, sample width ``dt``,
    and a start time ``start_time``, with the envelope being evaluated as
    :math:`f(t) =` ``s[floor((t - start_time)/dt)]`` if ``t`` is in the interval with endpoints ``start_time``
   and ``start_time + dt * len(samples)``, and ``0.0`` otherwise.
```